server/lib/services/feed_service.dart

```diff
@@ -13,6 +13,7 @@ class FeedService {
 
   const FeedService(this._connection);
 
+  //if don't want replies in feed, then need to get rows from comment where parent_id is null
```

Comment:
Add a condition

`parent_id is null`

Reply:
`static const _randomFeedQuery =
      'select c.id, p.id as poll_id, c.comment, p.topic as poll_topic, '
      'p.emoji as poll_emoji '
      'from comment c tablesample SYSTEM_ROWS(10) **where parent_id = 0**'
      'join poll p on c.poll_id = p.id '
      'where p.end > @now and (select count(1) from vote v '
      'where v.user_id = @user_id and v.comment_id = c.id) = 0';`
I added the condition like this - not sure if this is where it's supposed to go. I'm not too familiar w psql, but this is where I think it'd probably go?